doc/reference/reference_lua/box_schema/user_create.rst

```diff
@@ -21,7 +21,7 @@ box.schema.user.create()
     * ``password`` (default = '') - string; the ``password`` = *password*
       specification is good because in a :ref:`URI <index-uri>`
       (Uniform Resource Identifier) it is usually illegal to include a
-      user-name without a password.
+      username without a password.
 
     .. NOTE::
```

Comment:
Wow, this surprised me a lot! 
Isn't it important enough to mention in the **Users** section of **Access control**?

<img width="1022" alt="image" src="https://github.com/tarantool/doc/assets/42832629/88de1469-acc0-48c7-840b-e1abfab5681b">

Reply:
This is also described in Limitations: https://www.tarantool.io/en/doc/latest/book/box/limitations/.
I'd keep it as is because it might be hard to maintain such reference information is multiple places.